### PR TITLE
Ds 2808

### PR DESCRIPTION
**Pull request by chschan**

### CODE REVIEW

File: R/chartutils.R [hunk 1]
```diff
@@ -81,7 +81,8 @@ checkMatrixNames <- function(x, assign.col.names = TRUE, use.annot = FALSE)
         rownames(x) <- t.seq
```
Comment (mwmclean) [inline R/chartutils.R]: Can you please document this function. What is `x`? What formats for `x` are supported? Are there weird edge cases (e.g. this bug, DS-2808) that need to be documented, so we don't look back at this code and say "why is this here?". What is being returned? It seems to be doing more than checking the names, i.e. updating them somehow and returning a new `x`.

Comment (chschan) [reply R/chartutils.R]: Added documentation in commit d354237
